staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -132,9 +132,10 @@ type Event struct {
 
 	// Annotations is an unstructured key value map stored with an audit event that may be set by
 	// plugins invoked in the request serving chain, including authentication, authorization and
-	// admission plugins. Keys should uniquely identify the informing component to avoid name
-	// collisions (e.g. podsecuritypolicy.admission.k8s.io/policy). Values should be short. Annotations
-	// are included in the Metadata level.
+	// admission plugins. But it does not contain the annotations of the submitted object.
```

Comment:
suggest "Note that these annotations are for the audit event, and do not contain the metadata.annotations of the submitted object."

Reply:
also update v1alpha1 and v1beta1 audit types?

Reply:
Done.